src/fragments/lib/auth/flutter/user_attributes/50_custom_attributes.mdx

```diff
@@ -0,0 +1,26 @@
+## Custom attributes
+Amplify Flutter supports [standard OIDC user attributes](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims) as well as custom attributes. Custom attributes can be instantiated via the custom attribute constructor:
```

Comment:
do we want to mention OIDC here - or mention the standard attributes from Cognito?

Reply:
They are the same, I believe.

Reply:
I think it would make sense to link to Cognito's attributes

Reply:
Oh I see - yeah, sure.

Reply:
I'm okay either way. Cognito links to OIDC as well: https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-attributes.html#cognito-user-pools-standard-attributes

Reply:
hmmm - good point. Let us keep it as is for now.